packages/idx/src/idx.ts

```diff
@@ -1575,23 +1575,25 @@ const angularJsService = (
         if (serviceIds && _.isArray(serviceIds)) {
             serviceIds.forEach(serviceId => {
                 if (Object.prototype.hasOwnProperty.call(session.services, serviceId)) {
-                    serviceList[session.services[serviceId].id] = {
+                    serviceList.push({
```

Comment:
The difference of `push` is that it gives duplicates, as long as that's intentional.

Reply:
`serviceList` is a fresh new array being created with push, so yeah